content/posts/android-tv-14.md

```diff
@@ -0,0 +1,110 @@
+
+Go to the [download](https://jellyfin.org/downloads/) page or read the [installing](https://jellyfin.org/docs/general/administration/installing.html) documentation to start updating.
+
+## Improved login
+
+It's now a lot easier to get started with Jellyfin on your television using Quick Connect. You no longer need to enter a password and instead you'll be prompted to open Jellyfin on another device, like your phone or computer to enter a 6 digit code. This way of authentication is an alternative to passwords while keeping your server secure.
+
+Another big change in the login part of the app, are the redesigned screens when adding a server or user. These screens are now easier to read and use by using bigger text and a better alignment for the on-screen keyboard. Google TV users will now see the keyboard on the right side of the television. This makes it possible to see connections errors without closing the keyboard.
+
+![Screenshot of the new login design](/images/posts/android-tv-14/login-1.png)
+
+## Support for external trailers
+
+The newly added support for external trailers allows the playback of trailers from YouTube, Vimeo and other video services. Previously only local trailers could be played. This feature is only available when a browser or video service app is installed to open the trailer.
+
+Unfortunately this feature has some limitations. Some popular apps, like YouTube, don't make it easy to go back to the Jellyfin app. And in some television platforms you might see an error telling you there is no app to open the trailer. If this happens you should check if the YouTube/Vimeo/other app is installed. For an optimal experience we recommend using local trailers that play inside the Jellyfin app.
+
+## Changes to the Live TV guide
+
+The Live TV guide got some long awaited fixes and design tweaks. These include, but are not limited to:
+
+- Fixed the incorrect sizing of channel images in some places
+- Fixed an issue where the guide could have some sizing problems with many channels
+- Fixed the "null" text when information was missing in a some places
+- Fixed guide sorting not always working
+- Removed the category images causing contrast issues
+- Restyled some buttons to use theme colors
+- Restyled the filter and options popups to the same design as other parts of the app
+
+## Improved browsing grid
+
+To make it easier to see a lot of items or just a few we have added two new image sizes in the grid view: smallest and extra large. Together with this change the sizing is now based on the actual size of your device, fixing some bugs on 4k displays. This might change the size you're used to a bit. We also fixed some issues with the vertical direction and now consider it a stable feature!
```

Comment:
```suggestion
To make it easier to see a lot of items or just a few we have added two new image sizes in the grid view: smallest and extra large. Together with this change the sizing is now based on the actual size of your device, fixing some bugs on 4K displays. This might change the size you're used to a bit. We also fixed some issues with the vertical direction and now consider it a stable feature!
```
And "vertical direction" = vertical layout?